### PR TITLE
Integrate open source wireless driver rtl8139

### PR DESCRIPTION
**Pull request by aahud**
This driver was acquired from the University of Valencia
ftp server. http://ftp.uv.es/pub/linux/rtl8139.c

Hopefully I can successfully integrate this open source code quickly and easily.

Signed-off-by: Aaron Hudson <aaron.hudson@student.nmt.edu>

### CODE REVIEW

File: sys/src/9/386/rtl8139.c [hunk 1]
```diff
@@ -0,0 +1,1418 @@
+/* rtl8139.c: A RealTek RTL8129/8139 Fast Ethernet driver for Linux. */
```
Comment (rminnich) [inline sys/src/9/386/rtl8139.c]: we already have an 8139 driver. And 8139 are not wireless. I'm not sure what this is?

File: sys/src/libacpi/acpica/acpiflags.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
 		        "-DACPI_DISASSEMBLER",
 		        "-DACPI_EXEC_APP",
 		        "-D__HARVEY__",
+                "-U__FreeBSD__",
```
Comment (rminnich) [inline sys/src/libacpi/acpica/acpiflags.json]: This should be in a separate commit.

**Comment by rminnich** [comment]
-2

Aaron, 2 things: we have a driver for the 8139; and the .json fix should be its own commit.

**Comment by aahud** [comment]
Partially an accident, I also have a driver for the 8180 on my machine. But I won't lie unaware of the 8139 support. I will get rid of this create a separate issue/commit for the BSD line if that hasn't been done already and we can go from there.
